### ISSUE TITLE
FileNotFoundException with daily filewriter

### ISSUE THREAD

**Issue by cornerman** [outr/scribe#59]
At midnight, I just got a FileNotFoundException in my application from logging. This came from a rotating filewriter `FileWriter.daily(nio = false)` and somehow the file was not there yet.

```
[ERROR] [05/11/2018 00:00:25.108] [server-akka.actor.default-dispatcher-8] [akka://server/user/$l] logs/app.2018-05-11.log
core java.nio.file.NoSuchFileException: logs/app.2018-05-11.log
core 	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:86)
core 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
core 	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
core 	at sun.nio.fs.UnixFileSystemProvider.isSameFile(UnixFileSystemProvider.java:338)
core 	at java.nio.file.Files.isSameFile(Files.java:1504)
core 	at scribe.writer.FileIOWriter.$anonfun$validate$1(FileIOWriter.scala:27)
core 	at scribe.writer.FileIOWriter.$anonfun$validate$1$adapted(FileIOWriter.scala:27)
core 	at scala.Option.exists(Option.scala:240)
core 	at scribe.writer.FileIOWriter.validate(FileIOWriter.scala:27)
core 	at scribe.writer.FileIOWriter.write(FileIOWriter.scala:18)
core 	at scribe.handler.SynchronousLogHandler.$anonfun$log$3(SynchronousLogHandler.scala:19)
core 	at scribe.handler.SynchronousLogHandler.$anonfun$log$3$adapted(SynchronousLogHandler.scala:18)
core 	at scala.Option.foreach(Option.scala:257)
core 	at scribe.handler.SynchronousLogHandler.log(SynchronousLogHandler.scala:18)
core 	at scribe.Logger.$anonfun$log$4(Logger.scala:39)
core 	at scribe.Logger.$anonfun$log$4$adapted(Logger.scala:39)
core 	at scala.collection.immutable.List.foreach(List.scala:389)
core 	at scribe.Logger.$anonfun$log$3(Logger.scala:39)
core 	at scribe.Logger.$anonfun$log$3$adapted(Logger.scala:38)
core 	at scala.Option.foreach(Option.scala:257)
core 	at scribe.Logger.log(Logger.scala:38)
core 	at scribe.Logger.$anonfun$log$6(Logger.scala:40)
core 	at scribe.Logger.$anonfun$log$6$adapted(Logger.scala:40)
core 	at scala.Option.foreach(Option.scala:257)
core 	at scribe.Logger.$anonfun$log$3(Logger.scala:40)
core 	at scribe.Logger.$anonfun$log$3$adapted(Logger.scala:38)
core 	at scala.Option.foreach(Option.scala:257)
core 	at scribe.Logger.log(Logger.scala:38)
core 	at scribe.package$.log(package.scala:7)
core 	at covenant.ws.api.ApiRequestHandler.onClientConnect(ApiRequestHandler.scala:24)
core 	at mycelium.server.ConnectedClient.connected(ConnectedClient.scala:62)
core 	at mycelium.server.ConnectedClient$$anonfun$receive$1.applyOrElse(ConnectedClient.scala:67)
core 	at akka.actor.Actor.aroundReceive(Actor.scala:517)
core 	at akka.actor.Actor.aroundReceive$(Actor.scala:515)
core 	at mycelium.server.ConnectedClient.aroundReceive(ConnectedClient.scala:20)
core 	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:588)
core 	at akka.actor.ActorCell.invoke(ActorCell.scala:557)
core 	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:258)
core 	at akka.dispatch.Mailbox.run(Mailbox.scala:225)
core 	at akka.dispatch.Mailbox.exec(Mailbox.scala:235)
core 	at akka.dispatch.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
core 	at akka.dispatch.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
core 	at akka.dispatch.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
core 	at akka.dispatch.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by darkfrog26** [comment]
What version of Scribe are you using?

**Comment by cornerman** [comment]
This is the latest release 2.3.4

**Comment by darkfrog26** [comment]
I just published `2.3.5-SNAPSHOT`.  If you have Sonatype Snapshots repo can you give it a try and see if that resolves your problem?  I was making an erroneous assumption about `Files.isSameFile` from their ambiguous docs.

**Comment by cornerman** [comment]
Looks good to me. Until now I did not have this issue again.
